Airports/EGLL/SMR/Geo.txt

```diff
@@ -2195,35 +2053,8 @@ N051.28.02.506 W000.27.20.036 N051.28.02.308 W000.27.20.251 smrLLTaxiway
 N051.28.02.308 W000.27.20.251 N051.28.02.118 W000.27.20.410 smrLLTaxiway
 N051.28.02.118 W000.27.20.410 N051.28.01.920 W000.27.20.501 smrLLTaxiway
 N051.28.01.920 W000.27.20.501 N051.28.01.722 W000.27.20.558 smrLLTaxiway
-;Taxiway_Centreline_L34
-N051.28.02.949 W000.27.29.988 N051.28.02.907 W000.27.29.520 smrLLTaxiway
-N051.28.02.907 W000.27.29.520 N051.28.02.800 W000.27.29.098 smrLLTaxiway
-N051.28.02.800 W000.27.29.098 N051.28.02.687 W000.27.28.790 smrLLTaxiway
-N051.28.02.687 W000.27.28.790 N051.28.02.509 W000.27.28.471 smrLLTaxiway
-N051.28.02.509 W000.27.28.471 N051.28.02.281 W000.27.28.197 smrLLTaxiway
-N051.28.02.281 W000.27.28.197 N051.28.02.054 W000.27.28.015 smrLLTaxiway
-N051.28.02.054 W000.27.28.015 N051.28.01.848 W000.27.27.923 smrLLTaxiway
```

Comment:
Should this have been removed?

Reply:
I can still see that in my Geo file though. I've re-added it.